### PR TITLE
fix(anytracker): remove patch

### PR DESCRIPTION
**Pull request by shervinkoushan**
Removes AnyTracker patch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,14 +148,6 @@ The official Patch bundle provided by ReVanced and the community.
 | `unlock-themes` | Unlocks all themes. | all |
 </details>
 
-### 📦 `com.shervinkoushan.anyTracker`
-<details>
```
Comment (oSumAtrIX) [inline README.md:152]: Please do not commit this file, it is autogenerated.

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

**Review by oSumAtrIX — APPROVED** [review]

**Comment by TheJeterLP** [comment]
Why has this been removed?

**Comment by oSumAtrIX** [comment]
Thus PR was merged on request. It may be subject to be reverted at some point.

**Comment by Canny1913** [comment]
> Why has this been removed?

Takedown was requested by app developer.

**Comment by nikhilbadyal** [comment]
> > Why has this been removed?
> 
> Takedown was requested by app developer.

Can goolang also raise a request like this and you will pull youtube down ?

**Comment by oSumAtrIX** [comment]
It would be a request which we would decide to accept or not based on the respective situation.

**Pull request merged**